platform/darwin/src/MGLNetworkConfiguration.m

```diff
@@ -1,8 +1,15 @@
-#import "MGLNetworkConfiguration.h"
+#import "MGLNetworkConfiguration_Private.h"
+
+
+NSString * const kMGLDownloadPerformanceEvent = @"mobile.performance_trace";
```

Comment:
I think this needs to be in `MMEConstants.m`? Which will require a telemetry SDK update. /cc @andrlee @rclee

Reply:
Yep, this is an event that will be used fairly widely. It belongs in `MMEConstants`.

Reply:
@julianrex It is certainly a good idea that said it should not block this PR, @rclee would you be able to cut a ticket for this ask?